### PR TITLE
Yorhodes/tx parse fixes

### PR DESCRIPTION
**Pull request by yorhodes**

### CODE REVIEW

File: airgap/client.go [hunk 1]
```diff
@@ -94,17 +95,21 @@ func (c *clientImpl) GenerateProofOfPossessionSignature(privateKey *ecdsa.Privat
 	return signature, err
 }
 
-var abiParsers = []func() (*abi.ABI, error){
-	contracts.ParseReleaseGoldABI,
-	contracts.ParseAccountsABI,
-	contracts.ParseLockedGoldABI,
-	contracts.ParseElectionABI,
+var abiParsers = map[string]func() (*abi.ABI, error){
+	ReleaseGold:                            contracts.ParseReleaseGoldABI,
```
Comment (pranaymohan) [inline airgap/client.go:99]: qq why not do `registry.ReleaseGoldContractID.String()`?

Comment (yorhodes) [reply airgap/client.go:99]: ReleaseGold doesn't exist in the registry because there is a deployment per beneficiary

Comment (pranaymohan) [reply airgap/client.go:99]: ah gotcha, no one canonical contract, so you have to hardcode the key

File: airgap/client.go [hunk 2]
```diff
@@ -94,17 +95,21 @@ func (c *clientImpl) GenerateProofOfPossessionSignature(privateKey *ecdsa.Privat
...
+	registry.AccountsContractID.String():   contracts.ParseAccountsABI,
+	registry.LockedGoldContractID.String(): contracts.ParseLockedGoldABI,
+	registry.ElectionContractID.String():   contracts.ParseElectionABI,
 }
 
 func parseMethodAndArgs(data []byte) (*CeloMethod, []interface{}, error) {
+	if len(data) == 0 {
```
Comment (mcortesi) [inline airgap/client.go:106]: should this return an error??

**Pull request merged**